src/compiler/rust.rs

```diff
@@ -607,13 +606,8 @@ impl RustupProxy {
     where
         P: AsRef<Path>,
     {
-        let filetime = fs::metadata(proxy_executable.as_ref())
-            .map(|attr| FileTime::from_last_modification_time(&attr))?;
         let proxy_executable = proxy_executable.as_ref().to_owned();
-        Ok(Self {
-            proxy_executable,
-            filetime,
-        })
+        Ok(Self { proxy_executable })
```

Comment:
Could you elaborate on this change?

Reply:
This is never read as far as rustc is involved, so I just removed this... hence why I asked above if we want to use this somewhere. In other codepaths we just ignore it and prepend it with underscore `_filetime` :thinking: